test/lexer/Lexer.test.js

```diff
@@ -422,4 +422,23 @@ describe("lexer", () => {
             ]);
         });
     });
+
+    describe("isToken", () => {
+        it("works", () => {
```

Comment:
Mind changing this to `"returns booleans"` or something?

```suggestion
        it("returns booleans", () => {
```